.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: go
+
+go:
+  - 1.9
```

Comment:
Nit: `"1.9.x"` (string + x-version)

Also, any reason why not 1.10 too? Is 1.9 the minimum compat?